pkg/virt-operator/resource/apply/instancetypes.go

```diff
@@ -72,6 +72,16 @@ func (r *Reconciler) createOrUpdateInstancetype(instancetype *instancetypev1beta
 }
 
 func (r *Reconciler) deleteInstancetypes() error {
+	foundInstancetype := false
+	for _, instancetype := range r.targetStrategy.Instancetypes() {
+		if _, exists, _ := r.stores.ClusterInstancetype.GetByKey(instancetype.Name); exists {
```

Comment:
I think we need to cover the case where an error is returned as we do, for example, for deployment
```
	if err != nil {
		return err
	}
```